CitationStyleLanguageSettingsForm.inc.php

```diff
@@ -73,6 +99,13 @@ public function fetch($request, $template = null, $display = false) {
 			$allDownloads[$style['id']] = $style['title'];
 		}
 
+		$allUserGroups = [];
+		$userGroupDao = DAORegistry::getDAO('UserGroupDAO'); /* @var $userGroupDao UserGroupDAO */
+		$userGroups = $userGroupDao->getByContextId($contextId);
+		while ($userGroup = $userGroups->next()) {
+			$allUserGroups[(int) $userGroup->getId()] = $userGroup->getLocalizedName();
+		}
```

Comment:
Instead of allowing journals to select from all user groups, lets limit it to only user groups with `ROLE_ID_AUTHOR`. Those are the only ones eligible to be shown in the citation anyway.